ci/pipelines/deploy.yml

```diff
@@ -0,0 +1,162 @@
+resources:
+  - name: govwifi-admin
+    type: git
+    source:
+      uri: "https://github.com/alphagov/govwifi-admin.git"
+      branch: "ci-concourse-deployment"
+
+  - name: ecr
+    # This needs to be docker-image, until registry-image supports ECR
```

Comment:
Still not entirely clear what this means.  ECR isn't supported?

Reply:
will remove the comment as it's more of a `TODO`.
Will add into trello to migrate from using `docker-image` (which is deprecated) to `registry-image` once the functionality we need (ECR support) is merged.

Reply:
Right ok